app/src/main/java/org/simple/clinic/home/patients/PatientsScreen.kt

```diff
@@ -213,10 +222,29 @@ open class PatientsScreen(context: Context, attrs: AttributeSet) : RelativeLayou
   }
 
   fun showSimpleVideo() {
+    //Hard-coding to show this simple video view exists because, as of now,
+    // we are not sure if we will have variations of this training video.
+    // We should improve this implementation later.
     showHomeScreenBackground(R.id.simpleVideoLayout)
+    simpleVideoImage.setOnClickListener {
+      openYouTubeLinkForSimpleVideo()
+    }
+    videoTitleText.setOnClickListener {
+      openYouTubeLinkForSimpleVideo()
+    }
   }
 
   fun showIllustration() {
     showHomeScreenBackground(homeIllustration.id)
   }
+
+  private fun openYouTubeLinkForSimpleVideo() {
+    val appIntent = Intent(Intent.ACTION_VIEW, Uri.parse("vnd.youtube:$youTubeVideoId"))
```

Comment:
We should resolve these intents using the `PackageManager` one after the other, in case we are unable to resolve the first intent. It's also not safe to assume the user will have access to the browser. Because with multiple user accounts, it is possible that the browser is not available.

It's okay to now solve for this case, but if the browser is not available we should at least log to sentry so that we can decide if we should handle this case later.